### DISCUSSION TITLE
Running Playbook on Unraid

### DISCUSSION THREAD

**Discussion by Sivivatu** [techno-tim/k3s-ansible#396] [Questions] [answered]
I currently use Unraid as my VM Hypervisor. I have manually created 5 Ubuntu nodes and want to deploy k3s on this environment. My problem is the script currently only seems to recognise proxmox and forces that process to happen. In my system, I want to use the existing VMs that have been created (rather than dynamically create the VMs).

Is this possible, and I'm just missing something, or will it not work with the current script, and I need to either make a new set of roles for Unraid or remove the proxmox roles from the playbook?

**Comment by timothystewart6** [comment, marked as answer]
This playbook will work on any machine (virtual or not) as long as you have ssh access to it!  Just follow this guide or video as is and it will walk you through everything. https://technotim.live/posts/k3s-etcd-ansible/

**Reply by Sivivatu** [reply]
That is the guide I was following, but when I run the site playbook (after configuring my inventory file and creating the group_vars), I get the error:

```
fatal: [192.168.100.10]: FAILED! => {"msg": "The conditional check 'proxmox_lxc_configure' failed. The error was: error while evaluating conditional (proxmox_lxc_configure): 'proxmox_lxc_configure' is undefined. 'proxmox_lxc_configure' is undefined\n\nThe error appears to be in '/home/fatpunk/k3s-ansible/roles/lxc/tasks/main.yml': line 2, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n---\n- name: Check for rc.local file\n  ^ here\n"}
```

To me, there appears to be proxmox check happening, but because I don't have proxmox, the check fails. 

It could be that there is another configuration that wasn't mentioned (maybe it changed since the video was published), or I'm just being blind.

**Reply by Sivivatu** [reply]
I have just noticed that in the group_vars I have commented out the ```proxmox_lxc_configure: false```  which is why the check was failing.
